rentpath/rprel
# [PL] [40] Create annotated tag before release

[Resolves Issue #40 ](https://github.com/rentpath/rprel/issues/40)

- added ability to create annotated tag prior to release
- updated build to use tar.gz as extension
- updated version and changelog
- updated timex to latest
- updated bypass
- updated Timex
- use `DateTime` instead of `Timex.Date`
- set logger to `:warn` on test
- added function to create tag
- added function to create tags ref
- fixed warnings
- display github error and fallback to defaults
- updated tests to reflect additional calls to fetch full commit sha
- fixed crash issue on compile by downgrading tzdata to 0.1.8
For more details, see:
https://github.com/bitwalker/timex/issues/86
